lib/Component.lua

```diff
@@ -177,6 +177,7 @@ end
 ]]
 function Component:getElementTraceback()
 	return self[InternalData].virtualNode.currentElement.source
+		or "<enable element tracebacks>"
```

Comment:
I think we want this to return nil if source isn't present, otherwise it's not feasible for consumers of `getElementTraceback` to actually tell if tracebacks are enabled.

Beyond that, I think that's out of the scope of this PR.